test/staking/DonationsStaking.test.ts

```diff
@@ -196,12 +207,40 @@ describe("DonationsStaking - DonationStaking contract that receives funds in ETH
     let stakeAmount = ethers.utils.parseEther("10");
     await dai["mint(address,uint256)"](donationsStaking.address, stakeAmount);
     const totalStakedBeforeStake = await donationsStaking.totalStaked();
-    let transaction = await (await donationsStaking.stakeDonations(0)).wait();
+    let transaction = await (await donationsStaking.stakeDonations()).wait();
     const totalStakedAfterStake = await donationsStaking.totalStaked();
     expect(totalStakedAfterStake.sub(totalStakedBeforeStake)).to.be.equal(
       stakeAmount
     );
   });
+  it("it should stake donations with ETH according to 0.3% of pool", async () => {
```

Comment:
need to add a test for withdraw, that it converts eth and eth stays in contract.
i think also current withdraw test isnt good because it didnt test that all eth+token was sent to avatar

Reply:
updated tests